qiskit_experiments/curve_analysis/curve_analysis.py

```diff
@@ -72,8 +71,6 @@ class CurveAnalysis(BaseAnalysis, ABC):
             - ``name``: Name of the curve. This is arbitrary data field, but should be unique.
             - ``plot_color``: String color representation of this series in the plot.
             - ``plot_symbol``: String formatter of the scatter of this series in the plot.
-            - ``plot_fit_uncertainty``: A Boolean signaling whether to plot fit uncertainty
```

Comment:
I think it would be good to add a plotting option for specifying the sigma regions to plot. So you can put in None for no std dev in plot, or [1, 3] for what you have currently hard coded here, or any list of (positive) floats.